src/scripts/main.js

```diff
@@ -63,6 +63,21 @@ numberButtons.forEach(number => {
   });
 });
 
+function disableMathOperators(){
+  const stagedOperation = document.querySelector('.operators');
+  console.log(stagedOperation)
+  stagedOperation.setAttribute("disabled", true)
+  console.log('disabled')
+}
```

Comment:
The `disabled` attribute is being applied in the `section`. To work properly it needs to be applied to each button within the `section`.
I suggest you select all `.operators button` and use a `forEach` to apply the attribute.

Same to `enableMathOperators()`.

Reply:
Changes applied as suggested and tested. I also covered the case when the clear button is pressed, re enabling buttons